app/assets/javascripts/templates/form.html

```diff
@@ -21,7 +21,7 @@
       Instructions
     </label>
     <div class="col-md-5">
-      <textarea name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
+      <textarea id="instructions" name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
```

Comment:
Ah, you are right. I avoid ids almost always, but it's obviously needed in this case.  Thanks for the PR!